core-ui/src/components/Extensibility/components/Text.js

```diff
@@ -1,12 +1,11 @@
-import { EMPTY_TEXT_PLACEHOLDER } from 'shared/constants';
+import { useGetPlaceholder } from 'components/Extensibility/helpers';
 
-export const Text = ({ value, schema }) => {
+export const Text = ({ value, schema, structure }) => {
+  const { emptyLeafPlaceholder } = useGetPlaceholder(structure);
   if (typeof value === 'object') {
     return JSON.stringify(value);
-  } else if (!value) {
-    return EMPTY_TEXT_PLACEHOLDER;
-  } else {
-    return value;
   }
+
+  return value || emptyLeafPlaceholder;
```

Comment:
Shouldn't we check if this value is null/undefined instead of having this condition? `0` is a value that we should display, not hide.